fulu/single_layer_aug.py

```diff
@@ -4,27 +4,38 @@
 import torch.nn as nn
 from torch.utils.data import TensorDataset, DataLoader
 from sklearn.preprocessing import StandardScaler
+from sklearn.metrics import mean_squared_error
 
 from fulu._base_aug import BaseAugmentation, add_log_lam
 
 
 class NNRegressor(nn.Module):
-    def __init__(self, n_inputs=1, n_hidden=10):
+    def __init__(self, n_inputs=1, n_hidden=10, activation='tanh'):
         super(NNRegressor, self).__init__()
+        
+        act = nn.Tanh()
+        if activation == 'tanh':
+            act = nn.Tanh()
+        elif activation == 'relu':
+            act = nn.ReLU()
+        elif activation == 'sigmoid':
+            act = nn.Sigmoid()
```

Comment:
Consider to add an "else" block and raise ValueError if activation function is unknown, something like
```python
else:
    raise ValueError('activation function {} is not supported'.format(activation))
```

Reply:
done